src/player.js

```diff
@@ -1736,6 +1736,27 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  /**
+   * The text track changed event object
+   * @param {FakeEvent} event - payload with text track
+   * @returns {void}
+   * @private
+   */
+  _onTextTrackAdded(event: FakeEvent): void {
+    const videoElement = this.getVideoElement();
+    const trackIndex = videoElement
```

Comment:
1. This event can be dispatched only if video element is exists, so I don't see a reason for the short-if statement.
2. Even if for some reason the video element isn't exists, `trackIndex` will get `false` value, so it can be boolean or a number? and why to call `getTextTracks` in that scenario? it will be redundant no?
So as I see it, or wrap the whole method logic with `if (videoElement) {...}` or remove the protection.

Reply:
trackIndex fixed to get -1 if no video element.
getTextTracks enter to if condition.
I don't see it as a problem to add this protection in inline if cause it relevant only for this check and next if will ignore it

Reply:
trackIndex fixed to get -1 if no video element.
getTextTracks enter to if condition.
I don't see it as a problem to add this protection in inline if cause it relevant only for this check and next if will ignore it.